AA555A0000000000.json

```diff
@@ -0,0 +1,19 @@
+
+{
+  "gateway_conf": {
+    "gateway_ID": "AA555A0000000000",
```

Comment:
That gateway ID doesn't look very normal... Did you generated it by hand?

Reply:
No, I used " git clone https://github.com/ttn-zh/ic880a-gateway.git ~/ic880a-gateway" to setup my Raspi. The Problem was, that  the script rebooted my Raspi, before I could note down  die EUI. So I tried  to figure it out after the system was up  again, but did not  know exactly where  to look. Then  I just started the program and took this Number: AA555A0000000000 from the screen, as ist looked like the EUI. 
Can you tell me, where I can look for the correct number? Ist there a config  file  or  a log  where the Number ist inclulded?

Reply:
Run again the install.sh script in ic880a-gateway code, it will tell you what the ID is.

Reply:
@df1qq so, did you manage to find it running the installer again as @lucaspnw suggested? If so, could you please update this pull request accordingly? Thanks!

Reply:
thanks für the Info. I did a new install of my raspi and posted a new pullrequest. This one can be closed.
Thanks for the help.
The problem was most likely an undervoltage, which made the first installation incomplete.